### PR TITLE
Fix setup to install transitive numpy

### PR DESCRIPTION
**Pull request by NickHilton**
**CHANGES**
- Install distutils numpy first
- Then install regular setup for package

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -12,18 +13,21 @@
 http://www.ucs.louisiana.edu/~kxk4695/statcalc/pois2pval.for
 """
 
-fortran_ext = Extension(
-    "poisson_etest.poisson_etest_fortran", sources=["lib/poisson_etest.f"]
-)
+if __name__ == "__main__":
+    from numpy.distutils.core import setup, Extension
```
Comment (josephlbarnett) [inline setup.py:17]: why a numpy specific distutils/setuptools implementation here?

Comment (NickHilton) [reply setup.py:17]: Its to allow the fortran compiler to work I think - This I took from the original repo. 

I also tried with regular `setuptools.setup` but it hit a fortran error

File: setup.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
-from setuptools import setup, Extension
+from setuptools import dist
+dist.Distribution().fetch_build_eggs(['numpy>=1.17'])
```
Comment (josephlbarnett) [inline setup.py]: maybe need the `<2` to be the same as the bottom?

**Review by josephlbarnett — APPROVED** [review]
ok!

**Pull request merged**